kotlinx-coroutines-core/common/src/Dispatchers.common.kt

```diff
@@ -78,4 +78,14 @@ public expect object Dispatchers {
      * the value of [CoroutineStart.UNDISPATCHED].
      */
     public val Unconfined: CoroutineDispatcher
+
+    /**
+     * Shuts down the dispatcher.
```

Comment:
Please explain the reasoning of adding the dispatcher (K/N memory management) here and also mention that this signature is present only in `native-mt` versions and not in regular coroutines to simplify KMPP users adoption

Reply:
Also, you can mention that it will be available in regular versions when #2558 is implemented